modules/cnf-du-management-pods.adoc

```diff
@@ -0,0 +1,356 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-management-pods.adoc_{context}"]
+
+= Management pods
+
+For the purposes of achieving 2-core (4 HT CPU) deployment of SNO, the set of pods that are considered _management_ are limited to:
```

Comment:
First thought: You said that you replaced SNO throughout--thank you.

Second thought: Maybe replace these two sentences with two or so different sentences that position user workloads as distinct from cluster management workloads.  Then, you can indicate that the following tables identify the namespaces and pods (overwhelmingly `Deployment` objects, btw) that can be restricted to a subset of the CPUs on a node by configuring workload partitioning.